project/BuildSettings.scala

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2012-2013 Snowplow Analytics Ltd
+ * Copyright 2012-2019 Snowplow Analytics Ltd
```

Comment:
There should be a 1:1:1 relationship between issue tracker, commit messages and `CHANGELOG`. This gives us better traceability in future. 

For example here we'll need:

issue: `Extend copyright to 2019 (#XX)`
commit message: `Extend copyright to 2019 (close #XX)`
CHANGELOG: `Extend copyright to 2019 (#XX)`

(So this will have to be a separate commit that only changes the copyright term.)